apps/articles/client/article.coffee

```diff
@@ -95,6 +101,27 @@ module.exports = class ArticleIndexView extends Backbone.View
       url: url
       description: description
 
+  setupPromotedContent: =>
+    if @article.get('channel_id') is sd.PC_ARTSY_CHANNEL
+      new Partner(id: @article.get('partner_ids')?[0]).fetch
+        error: => $('.articles-promoted').hide()
+        success: (partner) ->
+          new Profile(id: partner.get('default_profile_id')).fetch
+            error: => $('.articles-promoted').hide()
+            success: (profile) ->
+              $('#article-body-container').addClass('promoted').prepend promotedTemplate
+                name: partner.get('name')
+                href: profile.href()
+    else if @article.get('channel_id') is sd.PC_AUCTION_CHANNEL
+      new Sale(id: @article.get('auction_ids')?[0]).fetch
+        error: -> @$el('.articles-promoted').hide()
+        success: (sale) ->
+          $('.article-section-header').hide()
+          $('#article-body-container').addClass('promoted').prepend promotedTemplate
```

Comment:
So I think there is an opportunity to refactor this to be a bit easier to read (and process).

In both cases, you're adding a class to the body-container and prepending a template. It probably makes sense to pull this out and make it its own method on the view. Something like:

```coffeescript
renderPromotedTemplate: (name, href, type) ->
   $('#article-body-container').addClass('promoted').prepend promotedTemplate
      name: name
      href: href
      type: type
```

This makes it easier to test as well, you wouldn't have to "fake" the successful fetch. This isn't vital to address before merging, more just something to think about!

Also, it's good practice to scope any jQuery to the view (i.e. `@$('.article-section-header').hide()`). See http://backbonejs.org/#View-dollar